subprojects/docs/src/docs/userguide/publishingMaven.adoc

```diff
@@ -18,33 +18,33 @@
 
 [NOTE]
 ====
-This chapter describes the new <<feature_lifecycle,incubating>> Maven publishing support provided by the “`maven-publish`” plugin. Eventually this new publishing support will replace publishing via the `Upload` task.
+This chapter describes the new <<feature_lifecycle,incubating>> Maven publishing support provided by the Maven Publish Plugin. Eventually this new publishing support will replace publishing via the `Upload` task.
 
 If you are looking for documentation on the original Maven publishing support using the `Upload` task please see <<artifact_management>>.
 ====
 
-This chapter describes how to publish build artifacts to an http://maven.apache.org/[Apache Maven] Repository. A module published to a Maven repository can be consumed by Maven, Gradle (see <<declaring_dependencies>>) and other tools that understand the Maven repository format.
+This chapter describes how to publish build artifacts to an http://maven.apache.org/[Apache Maven] repository. A module published to a Maven repository can be consumed by Maven, Gradle (see <<declaring_dependencies>>) and other tools that understand the Maven repository format.
 
 
 [[sec:the_mavenpublish_plugin]]
-=== The “`maven-publish`” Plugin
+=== The Maven Publish Plugin
 
-The ability to publish in the Maven format is provided by the “`maven-publish`” plugin.
+The ability to publish in the Maven format is provided by the Maven Publish Plugin.
 
-The “`publishing`” plugin creates an extension on the project named “`publishing`” of type api:org.gradle.api.publish.PublishingExtension[]. This extension provides a container of named publications and a container of named repositories. The “`maven-publish`” plugin works with api:org.gradle.api.publish.maven.MavenPublication[] publications and api:org.gradle.api.artifacts.repositories.MavenArtifactRepository[] repositories.
+The `publishing` plugin creates an extension on the project named `publishing` of type api:org.gradle.api.publish.PublishingExtension[]. This extension provides a container of named publications and a container of named repositories. The Maven Publish Plugin works with api:org.gradle.api.publish.maven.MavenPublication[] publications and api:org.gradle.api.artifacts.repositories.MavenArtifactRepository[] repositories.
 
 ++++
-<sample id="publishing_maven:apply_plugin" dir="maven-publish/quickstart" title="Applying the 'maven-publish' plugin">
+<sample id="publishing_maven:apply_plugin" dir="maven-publish/quickstart" title="Applying the Maven Publish Plugin">
     <sourcefile file="build.gradle" snippet="use-plugin"/>
 </sample>
 ++++
 
-Applying the “`maven-publish`” plugin does the following:
+Applying the `maven-publish` plugin does the following:
```

Comment:
This should be "Maven Publish Plugin". Unless you're referring to the actual code that applies the plugin, normal font applies.